README.md

```diff
@@ -96,9 +96,14 @@ response = contract.send_func(tx: tx, private_key: private_key, method: :transfe
 * rename top module name to `CITA`
 
 ### v0.21.0
+
 * add `recover` option for decode transaction, for CITA v0.21 provide `from` in `getTransaction` rpc call
 * rename protobuf `Crypto` enum
 
+### v0.22.0
```

Comment:
Can we re-organize the Changelog to make newest entries on top?

Reply:
Done